cloud/aws.go

```diff
@@ -227,17 +228,98 @@ func (m *awsResourceManager) BucketsPerAccount() map[string][]Bucket {
 						tags = convertAWSS3Tags(buTags.TagSet)
 					}
 
-					var count, size int64
-					var lastMod time.Time
+					cw := cloudwatch.New(sess, &aws.Config{Region: aws.String(region)})
```

Comment:
@AlexMichaelJonesNC think I forgot a `Credentials: creds` here. When I tested locally I removed a lot of `Credentials: creds` on (line 219/220 for example) because I could not assume role, but then I reverted all those changes I made. this one I didn't catch because it never had a `Credentials: creds`